doc/admin-guide/files/records.config.en.rst

```diff
@@ -4096,7 +4103,17 @@ HTTP/2 Configuration
    :reloadable:
    :units: bytes
 
-   The initial window size for inbound connections.
+   The initial window size for inbound connection streams.
+
+.. ts:cv:: CONFIG proxy.config.http2.session_initial_window_size_in INT 0
```

Comment:
This is #8203, right? This is obviously for incoming connections and should be merged separately. I was -0 on the PR. Take a look at the discussion on #8199 as well.

Reply:
Yes, correct. This PR pulls in that change. I agree that it's a good idea to keep it separate. I'll re-open #8203.

Reply:
Since #8203 has conflicts, I opened a new PR wherein I can resolve the conflicts: #9085.